core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/temporary/TemporaryDatabaseConnectionServiceImpl.java

```diff
@@ -127,4 +127,9 @@ public PreparedStatement createPreparedStatement(TemporaryDatabaseConnection dat
     public Class<TemporaryDatabaseConnection> appliesTo() {
         return TemporaryDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(TemporaryDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class